staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -223,37 +294,51 @@ func NodeUID(nodeUID types.UID) Option {
 	}
 }
 
+// Serialize overrides whether the helper serializes the prepare and unprepare
+// calls. The default is to serialize.
+//
+// A DRA driver can opt out of that to speed up parallel processing, but then
+// must handle concurrency itself.
+func Serialize(enabled bool) Option {
+	return func(o *options) error {
+		o.serialize = enabled
+		return nil
+	}
+}
+
 type options struct {
 	logger                     klog.Logger
 	grpcVerbosity              int
 	driverName                 string
 	nodeName                   string
 	nodeUID                    types.UID
-	draEndpoint                endpoint
-	draAddress                 string
 	pluginRegistrationEndpoint endpoint
+	pluginDataDirectoryPath    string
+	draEndpointListen          func(ctx context.Context, path string) (net.Listener, error)
 	unaryInterceptors          []grpc.UnaryServerInterceptor
 	streamInterceptors         []grpc.StreamServerInterceptor
 	kubeClient                 kubernetes.Interface
-
-	nodeV1alpha4 bool
-	nodeV1beta1  bool
+	serialize                  bool
+	nodeV1beta1                bool
 }
 
-// draPlugin combines the kubelet registration service and the DRA node plugin
-// service.
-type draPlugin struct {
+// Helper combines the kubelet registration service and the DRA node plugin
```

Comment:
Is there a less generic name than `Helper`?

Reply:
Suggestions? :sweat_smile: 

Remember that the full name is `kubeletplugin.Helper`. That is a bit more descriptive.

We could rename to "Adapter" or "Bridge", but those are also generic.

Reply:
Is `kubeletplugin.RegistrationHelper` more accurate? (I assuming this does more than just registration though).

What about `kubeletplugin.DRAHelper` since DRA is not readable at the callsite at all?

Reply:
`RegistrationHelper` is less accurate because it also implements the DRA plugin gRPC API.

`DRAHelper` may help, but is at the wrong place. A better solution is:
```
import drakubeletplugin "k8s.io/dynamic-resource-allocation/kubeletplugin"
...
...  drakubeletplugin.Helper ...
```

Reply:
Yes, that does look better.

l'm not going to block on this name though.